.travis.yml

```diff
@@ -1,15 +1,29 @@
 language: erlang
 
-otp_release:
-  - 18.0
-  - 18.1
-  - 18.2
-  - 18.3
-  - 19.0
-  - 19.1
-  - 19.2
+matrix:
+  include:
+    - os: linux
+      otp_release: 18.0
+    - os: linux
+      otp_release: 18.1
+    - os: linux
+      otp_release: 18.2
+    - os: linux
+      otp_release: 18.3
+    - os: linux
+      otp_release: 19.0
+    - os: linux
+      otp_release: 19.1
+    - os: linux
+      otp_release: 19.2
+    - os: osx
+      language: generic
 
-sudo: false
```

Comment:
actually, it turns out travis uses the date the repo was added to travis to determine the default, so `sudo: false` is required. i'll fix it